YacaVoiceProjekt/YacaVoiceServer.cs

```diff
@@ -105,11 +108,11 @@ public class YacaVoiceServer : Script
         #region Propertys
         public static YacaVoiceServer Instance { get; private set; }
         public int maxRadioChannels { get; private set; } = 9;
-        public string UNIQUE_SERVER_ID { get; private set; } = "abcdefg ServerUniqueIdentifier";
+        public string UNIQUE_SERVER_ID { get; private set; } = "";
         public int CHANNEL_ID { get; private set; } = 2; // Ingame TS Channel ID
         public string CHANNEL_PASSWORD { get; private set; } = "";
         public int DEFAULT_CHANNEL_ID { get; private set; } = 1; // AFK / Eingangshalle
-        public bool USE_WHISPER { get; private set; } = false;
+        public bool USE_WHISPER { get; private set; } = false; // Und das wird auch immer so bleiben
```

Comment:
kommentar sollte in der public ressource weg

Reply:
xD